go/teams/teams.go

```diff
@@ -287,7 +287,7 @@ func (t *Team) Rotate(ctx context.Context) error {
 	section.PerTeamKey = perTeamKeySection
 
 	// post the change to the server
-	if err := t.postChangeItem(ctx, section, secretBoxes, libkb.LinkTypeRotateKey, nil, nil); err != nil {
+	if err := t.postChangeItem(ctx, section, secretBoxes, libkb.LinkTypeRotateKey, nil, nil, false); err != nil {
```

Comment:
in order to accommodate the `permanence` option I added it as a parameter to `postChangeItem`, where it only matters if the link type is `libkb.LinkTypeLeave`. Should it be a `&bool` instead of a `bool` so these other methods can pass nil instead?